mbu/module1/src/main/java/com/epam/jmp/fileshare/exceptions/FileShareException.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.jmp.fileshare.exceptions;
+
+/**
+ * Created by nbuny on 21.07.2016.
+ */
+public class FileShareException extends Exception {
```

Comment:
The serializable class FileShareException does not declare a static final serialVersionUID field of type long